website/content/docs/secrets/pki/est.mdx

```diff
@@ -0,0 +1,181 @@
+
+Some clients allow a single label, sometimes referred to as `additional path segment`,
+to accommodate different issuers. This label will be added to the path after
+the est path such as `https://<hostname>:<port>/.well-known/est/<label>/`.
+
+To provide different restrictions around usage (defaults, an issuer or role),
+for EST protocol endpoints, a path policy is associated with the EST
+label.
+
+@include 'pki-est-default-policy.mdx'
+
+Within the Vault [EST configuration API](/vault/api-docs/secret/pki#set-est-configuration), a PKI
+mount can be specified as the default mount by enabling [default_mount](/vault/api-docs/secret/pki#default_mount)
+to true, or provide a mapping of a label within [label_to_path_policy](http://localhost:3000/vault/api-docs/secret/pki#label_to_path_policy)
+
+As an example of a complete EST configuration, that would enable the pki mount
+to register the .well-known/est default label, along with two additional labels
+of test-label and sign-all.
+
+The test-label would use the existing est-clients PKI role for restrictions and defaults,
+leveraging the issuer specified within the role. The other two labels, default and sign-all will
+leverage a sign-verbatim type role, allowing any identifier to be issued using the default
+issuer.
+
+```shell-session
+vault write pki/config/est -<<EOC
+{
+  "enabled": true,
+  "default_mount": true,
+  "default_path_policy": "sign-verbatim",
+  "label_to_path_policy": {
+    "test-label": "role:est-clients",
+    "sign-all": "sign-verbatim"
+  },
+  "authenticators": {
+    "cert": {
+      "accessor": "auth_cert_4088ac2d"
+    },
+    "userpass": {
+      "accessor": "auth_userpass_e2f4f6d5"
+    }
+  }
+}
+EOC
+```
+
+## Limitations
+
+### EST API Support
+
+The initial implementation covers solely the required API endpoints of the EST protocol.
+The following optional features from the specification are not currently supported.
+
+ - [Full CMC](https://datatracker.ietf.org/doc/html/rfc7030#section-4.3)
+ - [Server-side key generation](https://datatracker.ietf.org/doc/html/rfc7030#section-4.4)
+ - [CSR attribute endpoints](https://datatracker.ietf.org/doc/html/rfc7030#section-4.5)
+
+### Well Known redirections
+
+The EST configuration parameters `default_mount` and/or `label_to_path_policy` can be used to register
+paths within the .well-known path space. The following limitations apply:
+
+ - Only a single PKI mount, across all namespaces, can be enabled as the `default_mount`.
+ - Labels within `label_to_path_policy` must also be unique across all PKI mounts regardless of namespace.
+ - Care must be taken if enabling EST on a [local](/vault/docs/commands/secrets/enable#local) PKI mount on
+   performance secondary clusters. `default_mount` nor labels within `label_to_path_policy` must not conflict
+   with other replicated PKI mounts, depending on the ordering of a configuration update the registered label
```

Comment:
This any better?

```
 - Care must be taken if enabling EST on a [local](/vault/docs/commands/secrets/enable#local) PKI mount on
   performance secondary clusters. Vault cannot guarantee the configured EST labels do
   not conflict across different PKI mounts in this use-case. This can lead to
   different issuers being used across clusters for the same EST labels.
```